airbyte-cdk/python/airbyte_cdk/sources/declarative/schema/json_file_schema_loader.py

```diff
@@ -1,17 +1,18 @@
 #
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
-
 import json
 import pkgutil
 import sys
 from dataclasses import InitVar, dataclass, field
 from typing import Any, Mapping, Union
 
+from dataclasses_jsonschema import JsonSchemaMixin
```

Comment:
did autoformatting drag this up into it's own block? should it be below all the `airbyte-cdk` dependencies